apps/block_scout_web/lib/block_scout_web/controllers/address_controller.ex

```diff
@@ -60,27 +59,38 @@ defmodule BlockScoutWeb.AddressController do
   def index(conn, _params) do
     render(conn, "index.html",
       current_path: current_path(conn),
-      address_count: Chain.count_addresses_from_cache()
+      address_count: Chain.address_estimated_count()
     )
   end
 
   def show(conn, %{"id" => id}) do
     redirect(conn, to: address_transaction_path(conn, :index, id))
   end
 
-  def transaction_and_validation_count(%Hash{byte_count: unquote(Hash.Address.byte_count())} = address_hash) do
+  def address_counters(conn, %{"id" => address_hash_string}) do
+    with {:ok, address_hash} <- Chain.string_to_address_hash(address_hash_string),
```

Comment:
I get some errors here. Fixed them by changing to
```
    with {:ok, address_hash} <- Chain.string_to_address_hash(address_hash_string) do
      {transaction_count, validation_count} = transaction_and_validation_count(address_hash)
```

Reply:
fixed. Changed to `case` instead of `with`  (Credo recommendation)